packages/rln/src/contract/rln_light_contract.ts

```diff
@@ -0,0 +1,725 @@
+import { Logger } from "@waku/utils";
```

Comment:
this is exact copy of `rln_contract.ts` but without `rln instance`
I think we should extract base contract implementation and re-use, otherwise it's too much of a debt to take

Reply:
https://github.com/waku-org/js-waku/pull/2295/commits/9f8fd15ba9fbb216e24d5cec850434a50d1f721d